librz/core/cconfig.c

```diff
@@ -1737,6 +1737,180 @@ RZ_API bool rz_core_esil_cmd(RzAnalysisEsil *esil, const char *cmd, ut64 a1, ut6
 	return false;
 }
 
+static void eval_string(RzConfig *cfg, char *name) {
+	if (!*name) {
+		return;
+	}
+	char *eq = strchr(name, '=');
+	if (eq) {
+		*eq++ = 0;
+		rz_str_trim(name);
+		rz_str_trim(eq);
+		if (*name) {
+			(void)rz_config_set(cfg, name, eq);
+		}
+	} else {
+		if (rz_str_endswith(name, ".")) {
+			RzCmdStateOutput state = { 0 };
+			rz_cmd_state_output_init(&state, RZ_OUTPUT_MODE_QUIET);
+			rz_core_config_print_all(cfg, NULL, &state);
+			rz_cmd_state_output_print(&state);
+			rz_cmd_state_output_fini(&state);
+		} else {
+			const char *v = rz_config_get(cfg, name);
+			if (v) {
+				rz_cons_printf("%s\n", v);
+			} else {
+				rz_cons_printf("Invalid config key %s\n", name);
+			}
+		}
+	}
+}
+
+RZ_API bool rz_core_config_eval_and_print(RzCore *core, const char *str, bool many) {
+	rz_return_val_if_fail(core && str, false);
+
+	char *s = rz_str_trim_dup(str);
```

Comment:
this is not needed. let the caller do the trimming, if necessary (but it should not be).